Dockerfile.worker

```diff
@@ -0,0 +1,17 @@
+FROM python:2.7-stretch
+MAINTAINER Viktor Petersson <vpetersson@screenly.io>
```

Comment:
This is a new file. You'd be the maintainer ;)

Reply:
Got it :)